test/testdata/persistence/workflow/by_service/02-sonataflow_platform.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Apache Software Foundation (ASF)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: sonataflow-platform
+spec:
+  build:
+    template:
+      buildArgs:
+      - name: QUARKUS_EXTENSIONS
+        value: org.kie.kogito:kogito-addons-quarkus-jobs-knative-eventing:999-SNAPSHOT,org.kie.kogito:kogito-addons-quarkus-persistence-jdbc:999-SNAPSHOT,io.quarkus:quarkus-jdbc-postgresql:3.2.9.Final,io.quarkus:quarkus-agroal:3.2.9.Final
```

Comment:
@wmedvede I think it's worth adding those to the images, so we keep them in the cache to facilitate persistence use cases. wdyt?

Reply:
I think we need to assess here, because the persistence is a sort of all or nothing. I mean, if we add the org.kie.kogito:kogito-addons-quarkus-persistence-jdbc:999-SNAPSHOT to the swf-builder, then, every w the workflow must use persistence, otherwise we'll have strart-up failures.

O think we have the following scenarios:

1) The images is produced externally:
The workflow must come already built with the proper extensions incorporated in the produced image, and users might decide in a per workflow basis to include the persistence related addons. Finally the operator just relies on the workflow CRD con connect with the DB.

2) The Image is produced by the operator managed Kaniko or OpenShift based builds, here

But we are dealing with the persistence as an optional thing, so, it could happen the a workflow CRD with no configured persistence might come.